plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb

```diff
@@ -0,0 +1,49 @@
+module SamsonSlackWebhooks
+  class SlackWebhooksService
+    # users in the format jquery mentions input needs
+    # see _notify_buddy_box.html.erb
+    def users
+      Rails.cache.fetch(:slack_users, expires_in: 5.minutes, race_condition_ttl: 5) do
+        if slack_api_token
+          begin
+            body = JSON.parse(Faraday.post("https://slack.com/api/users.list", token: slack_api_token).body)
+            if body['ok']
+              body['members'].map do |user|
+                {
+                  id: user['id'],
+                  name: user['name'],
+                  avatar: user['profile']['image_48'],
+                  type: 'contact'
+                }
+              end
+            else
+              Rails.logger.error("Error fetching slack users: #{body['error']}")
```

Comment:
It's not explicitly documented. But it seems to imply that the api would always return 200. https://api.slack.com/web#basics